tests/helpers.bash

```diff
@@ -235,6 +235,17 @@ function wait_for_running_pod {
     set -x
 }
 
+function wait_for_no_pods {
+  set +x
+  namespace=${2:-default}
+  echo "Waiting for no pods to be Running in namespace ${namespace}"
+  kubectl get pods -n ${namespace} -o wide
+  while [[ "$(kubectl get pods -n ${namespace} -o wide 2>&1 | grep -c 'No resources found')" -ne "1" ]] ; do
```

Comment:
line 240 should be `namespace=${1:-default}`, but I don't see the need for ${no_pods}. On line 243, I am checking that the number of lines containing "No resources found" is 1, which is what is shown when there are no pods running in a namespace.